app/assets/javascripts/licenses.js

```diff
@@ -0,0 +1,24 @@
+var Licenses = {
+    init: function () {
+        $j('[data-role="seek-license-select"]').change(Licenses.displayUrl);
+        $j('[data-role="seek-license-select"]').change();
+        $j('[data-role="seek-license-select"]').select2({ theme: 'bootstrap' });
+    },
+
+    displayUrl: function () {
+        var element = $j('option:selected', $j(this));
+        var link = $j('#license-url');
+        if (link.length) {
+            var block = link.parents('.license-url-block');
+
+            if (element.data('url')) {
+                block.show();
+            } else {
+                block.hide();
+            }
+
+            link.attr('href', element.data('url'));
+            link.html(element.data('url'));
+        }
```

Comment:
might want to look at https://select2.org/programmatic-control/retrieving-selections about retrieving selections, which also advices against reading the selected option. Probably also use `select2:select` as the event for when it has been selected.